p22/benches/benches.rs

```diff
@@ -0,0 +1,28 @@
+#![feature(test)]
+
+extern crate test;
+
+#[cfg(test)]
+mod benchmarks {
+    use test::Bencher;
+
+    use p22::fibonacci::{fibonacci_non_recursive, fibonacci_recursive};
+
+    #[bench]
+    fn non_recursive(b: &mut Bencher) {
+        b.iter(|| {
+            for i in 1..=30 {
+                fibonacci_non_recursive(std::hint::black_box(i));
```

Comment:
Usually you benchmark function with a single parameter. You then can introduce separate benchmarks functions for different parameters, e.g. `bench_non_recursive_10`, `bench_non_recursive_20`, `bench_non_recursive_30`, etc. Or you can use [`criterion`](https://docs.rs/criterion) to automate this.

Reply:
done